### DISCUSSION TITLE
How to hide ad between shorts?!

### DISCUSSION THREAD

**Discussion by HassanDalwi** [ReVanced/revanced-discussions#898] [Question] [answered]
Hi

How can I disable advertising between shorts ?

There's an ad appears every 5 shorts !!!

How can I disable this ad.

**Comment by reisxd** [comment]
You can't.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
You now can hide shorts ads, by turning on `Switch home with notifications` under `Settings -> General layout -> Navigation buttons`.

That setting spoofs the device to an Android auto device, and YT does not appear to show any video ads for that environment.
